doc/source/whatsnew/v1.4.3.rst

```diff
@@ -24,7 +24,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Include ``dropna`` in ``__repr__`` of a :class:`Grouper` (:issue:`46754`)
```

Comment:
I think we should target this for 1.5.0; can you move this note to that whatsnew.

Reply:
Done in [the commit](https://github.com/pandas-dev/pandas/pull/46754/commits/dcdb7c7403bd14032788c31c5dfe37f81b6188ad) let me know if that's not what you're looking for. (Also assuming pandas/you prefer pushing fixup commits instead of altering my original commits and force-pushing?)

Reply:
Yep - looks great. I find it best to push fixups rather than force-pushing; the former is easier to pickup on from previous reviews.